### DISCUSSION TITLE
TUF Gaming F15 use G-Helper BUG

### DISCUSSION THREAD

**Discussion by MBXlong** [seerge/g-helper#1494] [Q&A] [answered]
1,The GPU fan stalls after applying a custom fan curve or applying a power limit
2,During the game, after the battery is stolen to a certain extent, the battery will be charged first, resulting in insufficient CPU power 
  consumption and a drop in frame rate

**Comment by MBXlong** [comment]
Forgot to mention, the software version is G-Helper 0.129.0

**Comment by seerge** [comment, marked as answer]
@MBXlong hello

1. G-Helper doesn't control your fans https://github.com/seerge/g-helper#how-does-g-helper-control-my-fan-speeds
2. G-Helper doesn't control how your computer charges anyhow, it's controlled by your hardware / firmware

**Reply by MBXlong** [reply]
Thank you for your answer, but it's strange that I don't do this when using ARMOURY CRATE, I once played 10 hours of continuous games to steal the battery power, but after using G-helper, although the battery rarely steals more than 10%, but after playing for a while, you can observe a significant drop in power consumption, and my CPU GPU temperature does not exceed 75°C, maybe it may be a problem with the power plan?

**Reply by seerge** [reply]
@MBXlong in AC the only way to set custom fan curve / power limits is via Manual mode (which is actually Turbo under the hood + custom limits) so try to use Turbo as base mode in G-Helper as well :)

As for your power - it's not controlled anyhow by the app, and it's not possible to do that (it would be nonsense to control charging from software running in windows anyway) :)

**Comment by MBXlong** [comment]
It's okay, I just restarted Ultron and found out that it's actually Armoury Crate
It is also like this that customizing the GPU fan curve will cause the GPU fan to stop, sorry to bother you, thank you very much for your answer,
Could this be a problem with the computer itself? But I still can't understand why the CPU fan can be tuned but the GPU cannot, could it be because of the problem of disabling some ASUS services.

**Reply by seerge** [reply]
@MBXlong yes, most probably it's specifics of your BIOS. You may try older version of it, sometimes it helps

**Comment by MBXlong** [comment]
I found the reason for the fan, I have to be connected to the GPU fan in a separate direct connection
